### PR TITLE
[TTAHUB-2222] correct erroneous fei goals

### PR DESCRIPTION
**Pull request by hardwarehuman**
## Description of change

Corrects some goals erroneously created as FEI goals in Region 11. Six are simply updated but a seventh is merged, which is why this is so long.

Compare to https://github.com/HHS/Head-Start-TTADP/blob/main/src/migrations/20231017230000-merge-pseudo-fei-goals.js

## How to test

The 6 goals have their text updated and their root causes removed. 51671 has its objective merged into 40495

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2222

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

**Comment by thewatermethod** [comment]
@hardwarehuman might be worth merging `main` into this so it will pass the CI

**Review by thewatermethod — COMMENTED** [review]
Confirmed that the script makes the required changes specified in the Jira! Will approve after main is merged in, the migration will need to be renamed to deploy

File: src/migrations/20231229000000-fix-erroneous-fei-goals.js [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+      UNION SELECT 4,'relinked_objective_resources', COUNT(*) FROM relinked_objective_resources
+      UNION SELECT 5,'relinked_objective_files', COUNT(*) FROM relinked_objective_files
+      UNION SELECT 6,'deleted_objective_topics', COUNT(*) FROM deleted_objective_topics
+      UNION SELECT 7,'deleted_objective_resources', COUNT(*) FROM deleted_objective_resources
+      UNION SELECT 8,'deleted_objective_files', COUNT(*) FROM deleted_objective_files
+      UNION SELECT 9,'relinked_objectives', COUNT(*) FROM relinked_objectives
+      UNION SELECT 10,'updated_target_goals', COUNT(*) FROM updated_target_goals
+      UNION SELECT 11,'relinked_args', COUNT(*) FROM relinked_args
+      UNION SELECT 12,'deleted_args', COUNT(*) FROM deleted_args
+      UNION SELECT 13,'deleted_goals', COUNT(*) FROM deleted_goals
```
[the remark this answers is not in the document]
Comment (hardwarehuman) [reply src/migrations/20231229000000-fix-erroneous-fei-goals.js]: Good catch! Fixed

**Comment by thewatermethod** [comment]
@hardwarehuman I had to revert this so you'll need to make a new PR

**Pull request merged**